### PR TITLE
Update outdated email address (letters A and B)

### PR DESCRIPTION
**Pull request by baltpeter**
As it turns out, our data is rotting a lot faster than I
expected.

This is the first batch of manual updates based on a tool
first proposed and implemented by @mal-tee that I have now
taken over.
So far, I have only completed the results for companies with
slugs starting with A and B, and the tool currently only checks
the email addresses.

Clearly, this is an important problem that we will need to deal
with. The ultimate goal for the tool is to automatically run it
periodically and report its findings as issues on this repo. This
is however not implemented yet but I will prioritize working on it.
I will also manually handle the first batch of findings mysel as
we would otherwise be completely overwhelmed with issues.
Hopefully, this will become more manageable once we actually
start regularly checking everything.

### CODE REVIEW

**Comment by baltpeter** [comment]
@mal-tee Obviously, it will take me a while to go through the remaining problems. Nonetheless, I think it makes sense to start merging my changes incrementally.

Feel free to decide yourself how thoroughly you want to review the changes. I am mostly submitting them as PR to illustrate how severe the problem actually is (I am honestly quite shocked myself).

File: companies/arvato-infoscore.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "name": "Arvato infoscore GmbH",
     "address": "Rheinstraße 99\n76532 Baden-Baden\nDeutschland",
     "phone": "+49 7221 50400",
-    "email": "datenschutz@arvato-infoscore.de",
+    "email": "datenschutz-aig@arvato.com",
```
Comment (mal-tee) [inline companies/arvato-infoscore.json:12]: Please set suggested-transport-medium as well :D

**Review by mal-tee — CHANGES_REQUESTED** [review]
Wow, really that's a lot! Thank you!

I've limited my "review" to running my (slightly extended) semgrep-rule for s-t-m and found one case. There a plenty of other in the whole of `data`, but I think I will wait for you to finish keep-aliving the rest before I fix them.

**Comment by baltpeter** [comment]
Please squash when merging.

**Review by mal-tee — APPROVED** [review]
Thanks!

**Pull request merged**